### PR TITLE
Added wishlist

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
Screenshot (47)

### CODE REVIEW

File: Wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                <a href="" class="nav-link"><i class="material-icons"> add_shopping_cart</i></a>
+            </li>
+        </ul>
+    </header>
+
+    <h1 class="heading-text center-text">My Wishlist(1)</h1>
+
+    <hr>
+
+    <section class="wishlist-main-container">
```
Comment (mohitdhatrak) [inline Wishlist/wishlist.html]: maybe instead of section, main tag would be better as only 1 section is used, so main tag feels appropriate

**Review by mohitdhatrak — COMMENTED** [review]
Looks good overall, well done!

File: Wishlist/wishlist.html (hunk 1 above)
Comment (syed-shaida-hussain) [reply Wishlist/wishlist.html]: Ok , will do the changes. Thank you.

**Review by NeerajVodala — APPROVED** [review]
Looks good to me.
All the best.

**Pull request merged**